client/dwh_migration_client/batch_sql_translator.py

```diff
@@ -24,7 +24,7 @@
 from os.path import dirname, join
 from typing import Optional
 
-from google.cloud import bigquery_migration_v2
+from google.cloud import bigquery_migration_v2alpha
```

Comment:
from google.cloud import bigquery_migration_v2_alpha as bigquery_migration_v2

and don't edit any of the code below.

Reply:
Good idea!